src/app/shared/form/builder/ds-dynamic-form-ui/models/year-picker/year-picker.component.html

```diff
@@ -12,12 +12,12 @@
             [disabled]="model.disabled"
             [min]="minYear"
             [max]="maxYear"
-            [name]="'year'"
+            [name]="'year' | translate"
```

Comment:
If we use translate we need to add it in i18n files with a form like 
```
    [name] = "'form.year'| translate" 
```
for example.